vyper/semantics/types/utils.py

```diff
@@ -159,6 +167,13 @@ def get_type_from_annotation(
     BaseTypeDefinition
         Type definition object.
     """
+    if isinstance(node, vy_ast.Tuple):
+        values = node.elements
+        values_def: Tuple[BaseTypeDefinition, ...] = ()
+        for v in values:
+            values_def += (get_type_from_annotation(v, DataLocation.UNSET),)
```

Comment:
it's easier to work with the `tuple` constructor here, something like
```suggestion
        values = node.elements
        types = tuple(get_type_from_annotation(v, DataLocation.UNSET) for v in values)
```